src/wallet/walletdb.h

```diff
@@ -244,8 +244,8 @@ class CWalletDB
     //! Write wallet version
     bool WriteVersion(int nVersion);
 private:
-    CDB batch;
-    CWalletDBWrapper& m_dbw;
+    BerkeleyBatch batch;
```

Comment:
You could rename this to `m_batch` in this PR, although that may be out of scope.

Reply:
https://github.com/bitcoin/bitcoin/pull/11851#discussion_r175532351

> You could rename this to m_batch in this PR, although that may be out of scope.

Done in c4719633496d6be338d3c2ee3d64be613200ae82